models/enterprise_sonic/l3_acls/replaced_example_01.txt

```diff
@@ -0,0 +1,103 @@
+# Using replaced
+#
+# Before State:
+# -------------
+#
+# sonic# show running-configuration ip access-list
+# !
+# ip access-list test
+#  seq 1 permit ip host 192.168.1.2 any
+#  seq 2 permit icmp any host 192.168.1.2 type 8
+#  seq 3 deny 2 any any
+#  seq 4 deny ip any any vlan 10 remark Vlan10
+# !
+# ip access-list test1
+#  remark test_ip_acl
+#  seq 1 permit tcp 10.0.0.0/8 any
+#  seq 2 deny udp any 20.1.0.0/16 gt 1024
+#  seq 3 deny ip any any dscp 63
+# sonic#
+# sonic# show running-configuration ipv6 access-list
+# !
+# ipv6 access-list testv6
+#  seq 1 permit ipv6 host 192:168:1::2 any
+#  seq 2 deny icmpv6 any any
+# !
+# ipv6 access-list testv6-1
+#  remark test_ipv6_acl
+#  seq 1 permit ipv6 1000::/16 any dscp af22
+#  seq 2 deny tcp any 2000::1000:0/112 range 100 1000
+#  seq 3 permit tcp any any established
+#  seq 4 deny udp any eq 3000 any
+# sonic#
+
+  - name: Replace device configuration of specified Layer 3 ACLs with provided configuration
+    dellemc.enterprise_sonic.sonic_l3_acls:
+      config:
+        - address_family: 'ipv4'
+          acls:
+          - name: 'test2'
+            rules:
+            - sequence_num: 1
+              action: 'permit'
+              protocol:
+                name: 'tcp'
+              source:
+                prefix: '192.168.1.0/24'
+              destination:
+                any: true
+        - address_family: 'ipv6'
+          acls:
+          - name: 'testv6'
```

Comment:
The operation of the "replaced" state could be more clearly shown (and differentiated from "merged" state) if different sequence numbers from the previously configured ones were used for the modified "testv6" ACL group in this example playbook. For example, if sequence numbers 3 and 4 are used instead of 1 and 2, the "after" state would show that instead of adding two new sequence number ACLs to the existing ones, the old ACLs were "replaced" by the specified contents for the new ones.

Reply:
As suggested, have updated the replaced example to show removal of old rules and addition of new ones for the specified ACL.